src/screens/NewsDetails.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import NewsDetailsHeader from '../components/NewsDetails/Header';
+import NewsDetailsBody from '../components/NewsDetails/Body';
```

Comment:
It would be better if you add index.js in each directory and import it like
`../components`